x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/BooleanArrayVector.java

```diff
@@ -23,7 +23,9 @@ final class BooleanArrayVector extends AbstractVector implements BooleanVector {
 
     static final long BASE_RAM_BYTES_USED = RamUsageEstimator.shallowSizeOfInstance(BooleanArrayVector.class)
         // TODO: remove these extra bytes once `asBlock` returns a block with a separate reference to the vector.
-        + RamUsageEstimator.shallowSizeOfInstance(BooleanVectorBlock.class);
+        + RamUsageEstimator.shallowSizeOfInstance(BooleanVectorBlock.class)
+        // TODO: remove this if/when we account for memory used by Pages
+        + RamUsageEstimator.NUM_BYTES_OBJECT_ALIGNMENT;
```

Comment:
Strictly, this should be `NUM_BYTES_OBJECT_REF`, but it would need further alignment to object size, so practically it would make no difference to the cost of one more operation.